addons/l10n_hr_euro/data/account.account.template.csv

```diff
@@ -0,0 +1,553 @@
+hr_406000,406000,Energy used during production of goods and services,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_407000,407000,Energy - administration and sale department,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_408000,408000,Costs for samples,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_409000,409000,Standard cost adjustments,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_410000,410000,Telephone and transportation costs,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_411000,411000,External service costs used in goods production and providing services,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_412000,412000,Maintenance and securities services,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_413000,413000,Vehicle registration and permission costs,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_414000,414000,Leasing costs,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_415000,415000,"Promotion , sponsorship and fairs costs",expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_416000,416000,Intellectual services,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_417000,417000,Utilities and similar costs,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_418000,418000,Representation costs - hosting,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_419000,419000,Other external costs,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_420000,420000,Wages and salaries,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_421000,421000,Income tax and surtax costs,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_422000,422000,Contributions from salaries costs,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_423000,423000,Contributions on salaries costs,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_424000,424000,Gross salaries,expense,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_430000,430000,Amortisation of intangible assets,depreciation,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_431000,431000,"Depreciation of property, plant and equipment",depreciation,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_432000,432000,Depreciation of cars and other vehicles for personal transportation,depreciation,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_433000,433000,"Depreciation of property, plant and - management and selling department",depreciation,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_434000,434000,Depreciation surplus due to revaluation of assets,depreciation,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_435000,435000,"Depreciation of biological assets (vineyards, orchards, live stock)",depreciation,l10n_hr_euro.l10n_hr_euro_chart_template,False
+hr_436000,436000,Tax not-deductible depreciation,depreciation,l10n_hr_euro.l10n_hr_euro_chart_template,False
```

Comment:
please use `expense_depreciation` and not `depreciation`

don't forget to fix that in master too